### PR TITLE
privacy.md Learning Sequence

### PR DESCRIPTION
**Pull request by catenglaender**
Hi everyone,

As part of the ongoing project to map all privacy relevant information, this PR adds a privacy.md to the Learning Sequence.

Feedback and suggestions are welcome. Thanks!

All the best

### CODE REVIEW

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Learning Sequence Privacy
+
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong
+information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix
+via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+### General Information
+
+The Learning Sequence leads users through multiple pieces of content in a row. It reacts to the Learning Progress of its child objects and can summarize them into one combined Learning Progress. Therefor, the Learning Sequence mostly contains references but may display a snapshot of a user's learning path and hold the final evaluation as one resulting Learning Progress.
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Please rephrase to:
The Learning Sequene guides Users through a series of consecutive content pieces. It dynamically adapts to the Learning Progress of its child objects ...

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+As of the time of writing, Users do not need to be Members to access and progress in the Learning Sequence's content by default. If a child object is configured in a way that Users have access, the Users still have access when it's dropped in to the Learning Sequence. This may put the privacy and access concerns primarily onto the child objects. Only with further configuration and permission management the Learning Sequence can serve as a gate for controlling access. Alternatively, a Learning Sequence may be placed into a Course and use its features for access control.
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Please rephrase to:
At the time of writing, Users are not required to be Members to access and progress through the Learning Sequence's content by default. 
Sentence 2: [...] when the object is added to the Learning Sequence. 
Sentence 4: [...] However, with additional configuration and permission management, the Learning Sequence can function as a gatekeeper for access control.

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- **Participants (Members)**: Adding users as Participants (Members) to the Learning Sequence references their User object by ID. A User may not be a Member but still create and modify a Learning Progress status for any of the child objects they can access.
+- **Progress Data**: For the individual participants, the Learning Sequence stores:
+  * The time and date of the first access to the Learning Sequence.
+  * The time and date of the last access to the Learning Sequence.
+  * A reference to the item that the participant last worked on.
+  * State information about the pages in the Learning Sequence. The exact information that is stored here depends on the individual objects in the Learning Sequence.
+
+## Data being presented
+
+- **Learning Progress of child objects**: Members can see their Learning Progress for the individual child objects. Global Admins and any user with the corresponding permission (unset by default) can see the Learning Progress for all Sequence child objects of every individual Member. The Learning Progress list may display data of more Users beyond the Users who are currently subscribed Members.
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: User

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- **Last Visited Step:** Admins can see which child object any Member accessed last.
+- **Availability and Visibility**: Users might be able to see or access a child object depending on its settings regardless of their membership to the Learning Sequence. This reflects a user's roles and other privacy settings of the child object.
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Comma missing:
Availability and Visibility: Users may be able to view or access a child object depending on its settings, regardless of their membership in the Learning Sequence. 
Maybe rephrase to:
This depends on the User's roles and other privacy settings of the child object.

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- **Member Gallery**: If activated, all Members can see information about all other Members. However, users can uncheck individual pieces of information to not be shown publicly in their profile settings. This may cause some information to be rendered as "Undisclosed".
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Please rephrase to:
When enabled, all Members can see information about other Members. However, Users can uncheck individual pieces of information in their profile settings to prevent them from being shown publicly. This may result in some information being marked as "Undisclosed".

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  - removes the reference to their User object and consequently their entry from any Member list or gallery view
+  - does not cause their Learning Progress to be deleted
+- **Deleting a User from the system** also removes them from any member and learning progress list in the Learning Sequence.
+- **Deleting Learning Sequence**
+  - Deleting the Learning Sequence removes all references to Members, settings and states stored for the sequence itself, as well as all child objects and their data (according to their behavior on deletion).
+
+## Data being exported
+
+- Learning Sequence options (like how a "User may proceed" depending on Learning Progress cues) are exported referencing child objects by ID.
+- No User object data, not even reference IDs or Admin roles assignments are being exported.
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Please rephrase to: 
No User object data, including reference IDs or Admin role assignments, is exported.

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+## Summary
```
Comment (elyesaseidel) [inline components/ILIAS/LearningSequence/PRIVACY.md:55]: I really appreciate this tabular overiew! 💯

**Review by elyesaseidel — CHANGES_REQUESTED** [review]

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    - [Object](../ILIASObject/PRIVACY.md)
+    - [InfoScreen](../InfoScreen/PRIVACY.md)
+    - [Mail](../Mail/PRIVACY.md) for notifications
+    - [Tracking](../Tracking/PRIVACY.md) may control if a user can progress the sequence and the sequence has its own Learning Progress as well
+    - [Membership](../Membership/PRIVACY.md)
+- Most of the actual data storage is not done by the Learning Sequence object itself. Instead the services Membership and Tracking are heavily used.
+
+## Data being stored
+
+- **Participants (Members)**: Adding users as Participants (Members) to the Learning Sequence references their User object by ID. A User may not be a Member but still create and modify a Learning Progress status for any of the child objects they can access.
```
Comment (kseniialf) [inline components/ILIAS/LearningSequence/PRIVACY.md]: the 2nd sentence here, just for me to clarify: non-Members information (Learning Progress, ID etc.) is not stored here, is it? 
**If it's not:** you can delete this sentence (I think this information is already described in General Information, line 11,  and does not belong here)

File: components/ILIAS/LearningSequence/PRIVACY.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+- Learning Sequence options (like how a "User may proceed" depending on Learning Progress cues) are exported referencing child objects by ID.
+- No User object data, not even reference IDs or Admin roles assignments are being exported.
+
+## Summary
+
+| Data                       | Stored in DB                       | Shown to general user                      | Shown to admin                                 | Exported   | deletes when rmv Member | deletes when rmv Sequence |
+|----------------------------|------------------------------------|--------------------------------------------|------------------------------------------------|------------|-------------------------|---------------------------|
+| Participant (Member) Users | reference to by ID                 | if Member Gallery active                   | yes - name, login                              | no         | yes                     | yes                       |
+| Child objects              | reference to by ID                 | by default access regardless of membership | yes                                            | optionally | no                      | yes?                      |
```
Comment (kseniialf) [inline components/ILIAS/LearningSequence/PRIVACY.md]: Why "yes?" ?

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 1 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 2 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 3 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: the capitalization? changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 4 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 5 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 8 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: changed

File: components/ILIAS/LearningSequence/PRIVACY.md (hunk 9 above)
Comment (catenglaender) [reply components/ILIAS/LearningSequence/PRIVACY.md]: Needed to investigate a bit more closely. Deletion isn't optimal... traces are left in the database that can then (to my knowledge) never be seen in ILIAS ever again.... but they are still in the database.

**Comment by catenglaender** [comment]
Made all changes suggested by @kseniiahhu and @elyesaseidel :heavy_check_mark:
